docs/devcontainer/viz-tutorial.ipynb

```diff
@@ -0,0 +1,540 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Intro\n",
+    "\n",
+    "This assumes you have gone through the intro/mpi4py intro, and that you are comfortable generating ExaGO output.\n",
+    "\n",
+    "Now we will go through specifics to the vizualisation frontend, how it is installed, and a typical workflow.\n",
+    "\n",
+    "# Open Issues:\n",
+    "\n",
+    "- npm dependencies were pinned and have security issues\n",
+    "- requirements.txt has pinned versions and we cannot upgrade / reproduce\n",
+    "- OpenAI keys and other missing steps...\n",
+    "\n",
+    "# Generating file to visualize\n",
+    "\n",
+    "I don't know how to do gic file option, so we are going to use OPFLOW C++ CLI to do this..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ExaGO 1.6.0 built on Nov  9 2023\n"
+     ]
+    }
+   ],
+   "source": [
+    "!opflow --version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "TODO: Going to let Shri fill the details in here, as likely we don't want people churning large data sets in a small demo, and we can include pre-generated data I would hope..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!./opflow -netfile <netfile> -save_output -opflow_output_format JSON -gicfile <gicfilename>  "
```

Comment:
add some verbiage about how this will launch a webpage etc

Reply:
This line doesn't launch web page